### DISCUSSION TITLE
Y axis tick label cut on top in stacked bar chart

### DISCUSSION THREAD

**Discussion by kgaulin** [f5/unovis#354] [Q&A] [answered]
Hello, 

the last Y axis tick label is cut . Anyone have resolve this issue ?

![image](https://github.com/f5/unovis/assets/68933242/c945c609-e4ea-4390-ae31-ed413a0c894a)

**Comment by rokotyan** [comment, marked as answer]
@kgaulin You can fix it by adding a small `top` margin to the XY Container: https://unovis.dev/docs/containers/XY_Container/#margin
